Bloc-Memory/MGCard.class.st

```diff
@@ -12,66 +12,78 @@ When card is flipped on the face a player should see a symbol that is used to co
 ]]]
 "
 Class {
-	#name : #MGCard,
-	#superclass : #Object,
+	#name : 'MGCard',
+	#superclass : 'Object',
 	#instVars : [
 		'symbol',
 		'flipped',
 		'announcer'
 	],
-	#category : #'Bloc-Memory-Model'
+	#category : 'Bloc-Memory-Model',
+	#package : 'Bloc-Memory',
+	#tag : 'Model'
 }
 
-{ #category : #accessing }
+{ #category : 'accessing' }
 MGCard >> announcer [
 	"Return my announcer that is used to notify my observers"
 	
 	^ announcer ifNil: [ announcer := Announcer new ]
 ]
 
-{ #category : #actions }
+{ #category : 'actions' }
 MGCard >> disappear [
 	"Ask card to disapper"
 	
 	self notifyDisappear
 ]
 
-{ #category : #actions }
+{ #category : 'actions' }
 MGCard >> flip [
 	"Flip the card on the other side (toggle flipped state)"
 	
 	flipped := flipped not.
 	self notifyFlipped
 ]
 
-{ #category : #initialization }
+{ #category : 'initialization' }
 MGCard >> initialize [
 	super initialize.
 	flipped := false
 ]
 
-{ #category : #testing }
+{ #category : 'inspector extensions' }
+MGCard >> inspectCard: aBuilder [
+	<inspectorPresentationOrder: 1 title: 'card'>
+	| builder |
+	builder := StSimpleInspectorBuilder on: aBuilder.
+	builder key: 'card:' value: self symbol.
+
+	^ builder table
+]
```

Comment:
@rvillemeur would it fine for you to recreate the PR? I mean, create a new branch from this repo's master, and commit this code again (and create a new PR). this way, it will be exported by iceberg with the right Tonel version

Reply:
sure, will do